.vscode/settings.json

```diff
@@ -1,36 +0,0 @@
-{
```

Comment:
maybe don't delete this file? because everyone will need to copy&paste again. I would prefer only ignore it.

Reply:
Unfortunately, it is not possible. If you want to ignore a file, it has to be removed first. As described here: https://stackoverflow.com/a/25436481/8332763